src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -50,6 +50,21 @@ internal PublicClientApplicationBuilder WithUserTokenLegacyCachePersistenceForTe
             return this;
         }
 
+        /// <summary>
+        /// Configures the pca to use the recommended redirect Uri for the platform. 
+        /// If you are calling your API from a 
+        /// .NET Classic app - https://login.microsoftonline.com/common/oauth2/nativeclient
+        /// UWP - value of WebAuthenticationBroker.GetCurrentApplicationCallbackUri()
+        /// .NET Core - https://localhost
+        /// </summary>
```

Comment:
@jennyf19 : not for now, but when we support brokers, will it make sense to also have this method set the URI used for the broker scenario? (thinking ahead here)